core/server/index.js

```diff
@@ -108,6 +112,14 @@ function init(options) {
             });
 
         return new GhostServer(parentApp);
+    }).then(function (_ghostServer) {
+        ghostServer = _ghostServer;
+
+        // scheduling can trigger api requests, that's why we initialize the module after the ghost server creation
+        // scheduling module can create x schedulers with different adapters
+        return scheduling.init(_.extend(config.scheduling, {apiUrl: config.url + config.urlFor('api')}));
+    }).then(function () {
+        return ghostServer;
```

Comment:
This doesn't seem like the right place for this to me? 

Putting it here means that scheduling won't work for people using Ghost as an npm module, unless they also update their index.js files.

If I understand correctly the idea is that we need to initialise scheduling after Ghost is ready to start receiving requests, because the scheduler might decide to start sending API requests. However, would that not be better handled via an event listener on when the server has started?